app/views/tips/show.html.erb

```diff
@@ -1,10 +1,21 @@
-
-<div>
-  <h3><%= @tip.title %></h3>
-  <h4>by <%= @author.first_name %> <%= @author.last_name %></h4>
-  <p><%= @tip.body %></p>
-  <hr>
-  <button type="button" class="btn btn-info"><%= link_to "Edit", edit_tip_path(@tip) %></button>
-  <%= link_to "Delete tip", delete_tip_path(@tip), :method => 'get', :data => {:confirm => 'Are you sure you want to delete this post?'} %>
-
+<div class="container">
+ <div class = "row">
+   <div class="col-md-4">
+     <div class="img-thumbnail">
+       <%= image_tag @author.profile_image %>
+     </div>
+     <h3><strong> <%= @author.first_name %> <%= @author.last_name %></strong></h3>
+     <p><strong>Date Joined: </strong><%= (@author.created_at).strftime("%m/%d/%Y") %></p>
+     <p><strong>City: </strong> <%= @author.current_city %></p>
+   </div>
+   <div class="col-md-8">
+     <h3><%= @tip.title %></h3>
+     <p><%= @tip.body %></p>
+     <br>
+     <button type="button" class="btn btn-info"><%= link_to "Edit", edit_tip_path(@tip) %></button>
+     <%= link_to "Delete", delete_tip_path(@tip), :method => 'get', :data => {:confirm => 'Are you sure you want to delete this post?'} %>
```

Comment:
Remove `delete_tip` alias from routes. `delete_tip_path` describes the same path as `tip_path` (`tips#show`).